lib/beaker/cli.rb

```diff
@@ -13,11 +13,15 @@ def initialize
       @timestamp = Time.now
       @options_parser = Beaker::Options::Parser.new
       @options = @options_parser.parse_args
+      @attribution = @options_parser.attribution
       @logger = Beaker::Logger.new(@options)
       InParallel::InParallelExecutor.logger = @logger
       @options[:logger] = @logger
+      @attribution[:logger] = 'runtime'
```

Comment:
I don't like how I'm having to update the attribution hash every time the options hash is updated, so I was wondering what people's thoughts were on wrapping the update of the two hashes in a function something like:

def update_option(key, value, source)
  @options[key] = value
  @attribution[key] = source
end

So we could just replace the @options and @attribution update line pairs with:

update_option(:logger, @logger, 'runtime')